Example/APMAlertController/ViewController.swift

```diff
@@ -194,25 +185,55 @@ class ViewController: UIViewController {
     func alertTextTitleAttributedMessage(_ indexPath: IndexPath) {
         let title = "ABC123-45678-90"
         let attributedMessage = NSMutableAttributedString(string: "Lorem ipsum dolor sit amet,\nconsectetur adipiscing elit,\nsed do eiusmod tempor\nincididunt ut labore et dolore magna aliqua.")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/almas-dev/APMAlertController/blob/458898c990446c048fd2759812785a8e78fcda83/Example/APMAlertController/ViewController.swift#L187"></span>Line should be 150 characters or less: currently 195 characters</td>
    </tr>
  </tbody>
</table>